integration-tests/env.sh

```diff
@@ -28,8 +28,13 @@ function check_command_exist() {
 
 # Awlays execute at script located directory
 CWD=$(pwd)
-if [ x"$(basename "${0}")" = "xenv.sh" ]; then
-    cd "$(dirname "${0}")"
+# in non interactive shell
+# ${0} is -bash
+# TODO: replace this line, to make it work on non interactive shell
+if [ -f "${0}" ]; then 
```

Comment:
in non-interactive shell,
${0} is -bash
, basename throws an error

Reply:
this fix is not perfect, but it will make it compile

Reply:
what about replacing the lines 35 and 36 with:
```
    if [ x"$(basename -- "${0}")" = "xenv.sh" ]; then
        cd "$(dirname -- "${0}")"
    fi
```
?

Reply:
i will check

Reply:
it works fine!!
thanks